### PR TITLE
feat(fonts): font can be overwritten; adding move font

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/tag/constants.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ LICENSE file in the root directory of this source tree.
...
 
 export const STYLE = {
-  primary: DEFAULT_THEME.colors.primary400,
-  warning: DEFAULT_THEME.colors.warning400,
-  positive: DEFAULT_THEME.colors.positive400,
-  negative: DEFAULT_THEME.colors.negative400,
+  primary: LightTheme.colors.primary400,
+  warning: LightTheme.colors.warning400,
+  positive: LightTheme.colors.positive400,
+  negative: LightTheme.colors.negative400,
```
Comment (nadiia) [inline src/tag/constants.js]: Not related to this PR, but it needs to be changed to a string like `primary400 `, `warning400 `, etc, and then get the values from a `$theme` in a styled component. This way the LightTheme wouldn't be the default but whatever theme is passed to the Provider.

**Comment by schnerd** [comment]
Does this resolve #252?

**Comment by gergelyke** [comment]
yeah, I think so, if we extend it, so all values can be overwritten - I can do that, should we?

**Comment by schnerd** [comment]
I think it's fine to just resolve that issue now since fonts were the primary concern, and then if customers ask to override other things in the future we can re-evaluate how that fits into the current API.

**Comment by gergelyke** [comment]
makes sense

**Pull request merged**